tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
```

Comment:
Also https://bitbucket.org/hpk42/tox/issues/275/allow-running-commands-before-and-after